target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-wsq50.dts

```diff
@@ -0,0 +1,316 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/input/linux-event-codes.h>
+#include <dt-bindings/soc/qcom,tcsr.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	model = "Zyxel WSQ50";
+	compatible = "zyxel,wsq50";
+
+	aliases {
+		led-boot = &status_green;
+		led-boot-aux1 = &status_red;
+		led-boot-aux2 = &status_blue;
+		led-running = &status_green;
+		led-running-aux1 = &status_red;
+		led-running-aux2 = &status_blue;
+		led-failsafe = &status_red;
+		led-upgrade = &status_blue;
+	};
+
+	chosen {
+		bootargs-append = " clk_ignore_unused fstools_ignore_partname=1";
+		stdout-path = &blsp1_uart1;
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			gpios = <&tlmm 0x12 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+
+	soc {
+		rng@22000 {
+			status = "okay";
+		};
+
+		mdio@90000 {
+			status = "okay";
+		};
+
+		tcsr@1949000 {
+			compatible = "qcom,tcsr";
+			reg = <0x1949000 0x100>;
+			qcom,wifi_glb_cfg = <TCSR_WIFI_GLB_CFG>;
+		};
+
+		tcsr@194b000 {
+			compatible = "qcom,tcsr";
+			reg = <0x194b000 0x100>;
+			qcom,usb-hsphy-mode-select = <TCSR_USB_HSPHY_HOST_MODE>;
+		};
+
+		ess_tcsr@1953000 {
+			compatible = "qcom,tcsr";
+			reg = <0x1953000 0x1000>;
+			qcom,ess-interface-select = <TCSR_ESS_PSGMII>;
+		};
+
+		tcsr@1957000 {
+			compatible = "qcom,tcsr";
+			reg = <0x1957000 0x100>;
+			qcom,wifi_noc_memtype_m0_m2 = <TCSR_WIFI_NOC_MEMTYPE_M0_M2>;
+		};
+
+		usb2@60f8800 {
+			status = "okay";
+		};
+
+		usb3@8af8800 {
+			status = "okay";
+		};
+
+		watchdog@b017000 {
+			status = "okay";
+		};
+	};
+};
+
+&tlmm {
+	serial1_pins: serial1_pinmux {
+		mux {
+			pins = "gpio16", "gpio17";
+			function = "blsp_uart1";
+			bias-disable;
+		};
+	};
```

Comment:
Newline after

Reply:
This has been fixed - Added newline here and also cleaned up rest of DTS to match